### PR TITLE
Add support for accept/no_accept for instances.

### PR DESCRIPTION
**Pull request by naphta**
Coming to this role wanting to implement the keepalived configuration offered by [nginx plus](https://docs.nginx.com/nginx/admin-guide/high-availability/ha-keepalived-nodes/) and they have accept inside that configuration. Had a quick dig into the manpage and thought this should suffice.

### CODE REVIEW

File: templates/keepalived.conf.j2 [hunk 1]
```diff
@@ -93,6 +93,13 @@ vrrp_instance {{ name }} {
   {% if instance.dont_track_primary is defined and instance.dont_track_primary | bool %}
   dont_track_primary                               # Override VRRP RFC dont_track_primary default
   {% endif %}
+  {% if instance.accept is defined and instance.accept | bool %}
```
Comment (evrardjp) [inline templates/keepalived.conf.j2:96]:
```suggestion
  {% if instance.accept is defined %}
```

File: templates/keepalived.conf.j2 [hunk 2]
```diff
@@ -93,6 +93,13 @@ vrrp_instance {{ name }} {
...
+  {% if instance.accept %}
```
Comment (evrardjp) [inline templates/keepalived.conf.j2:97]:
```suggestion
  {% if instance.accept | bool %}
```

File: templates/keepalived.conf.j2 (hunk 1 above)
Comment (evrardjp) [reply templates/keepalived.conf.j2:96]: This way if the user doesn't define accept (the default), there is no change in the config file.
And if someone writes instance.accept false, would write no_accept appropriately.

**Review by evrardjp — CHANGES_REQUESTED** [review]
A few suggestions. Can you try those?

File: templates/keepalived.conf.j2 (hunk 1 above)
Comment (naphta) [reply templates/keepalived.conf.j2:96]: I've not tested the alternative (nor do I have an objection to it) but I'd have thought that my if which is checking if accept is defined and a boolean wouldn't pass in the case of `accept: `, thus not writing out `accept` or `no_accept` to the file.

**Pull request closed without merging**